content/docs/how-we-work/documentation.md

```diff
@@ -6,4 +6,4 @@ description: Documentation is highly underrated. It is probably the most underut
 
 # Documentation
 
-Documentation is highly underrated. It is probably the most underutilized, yet the most important factor of success. Think about any product or technology. The only thing that dictated its success long-term was its ease-of-use, and documentation is highly critical to achieve that.
+Documentation is highly underrated. It is probably the most underutilized, yet the most important factor of success. Think about any product or technology. The only thing that dictated its success long-term was its ease-of-use, and documentation is highly critical to achieve that. Documentation is one of the ways to show the [values of Axelerant](https://www.axelerant.com/about).
```

Comment:
@Yemaneberhan-Lemma, can you link to the specific value you mean here? Linking to the about page is confusing to me.